lib/snapshot-index.proto

```diff
@@ -0,0 +1,9 @@
+syntax="proto3";
+
```

Comment:
I was thinking so we could warn about different versions, but seems like the best practise is not to version them, so I guess having it in the Snapshot field makes more sense: http://stackoverflow.com/questions/8519381/how-does-protocol-buffer-handle-versioning